router/org.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this orgsitory.
+
+package router
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/go-vela/server/api"
+	"github.com/go-vela/server/router/middleware/org"
+)
+
+// orgHandlers is a function that extends the provided base router group
+// with the API handlers for org functionality.
+//
+// POST   /api/v1/orgs
```

Comment:
I'm not sure this list is entirely accurate because you don't call the repo handlers from within the org handlers.

I'm also wondering if this break convention a little bit the way it's laid out. I'm wondering if it should be 

`/api/v1/org/:org/repo/:repo/builds`

You also have `orgs` when I think you mean `org`

Have you thought about looking into creating the org table with this change? I think it would open us up to more future functionality that we have been needing.